enhancements/baremetal/baremetal-config-raid-and-bios.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Config-RAID-and-BIOS-for-Baremetal-IPI-deployments
+authors:
+  - "@fenggw-fnst"
+  - "@Hellcatlk"
+  - "@hs0210"
+  - "@zhouhao3"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-05-18
+last-updated: 2021-05-24
+status: provisional
+see-also:
+  - 
+replaces:
+  -
+superseded-by:
+  - https://github.com/metal3-io/baremetal-operator/pull/302
+---
+
+# Config RAID and BIOS for Baremetal IPI deployments
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add the support of RAID and BIOS configuration for baremetal IPI deployments.
+
+## Motivation
+
+With the new feature of [RAID configuration](https://github.com/metal3-io/baremetal-operator/pull/292) and the upcoming feature of [BIOS configuration](https://github.com/metal3-io/baremetal-operator/pull/302), openshift already has or will have the ability to configure RAID and BIOS for baremetal.
+
+However, these features only work with a running cluster, users cannot configure desired RAID and BIOS during IPI deployments.
+
+### Goals
+
+- Allow users to configure RAID and BIOS in ***install-config.yaml***.
```

Comment:
We try not to list the implementation being proposed as a goal, since we might receive feedback that leads to a different implementation. Here I think it would be more accurate to say that we want to allow users to manage the RAID and BIOS settings for control plane hosts during installation, since worker nodes are already managed by the BMO (even when creating the cluster).

Reply:
Thanks for your review. Done.